fabric-biome-api-v1/src/main/java/net/fabricmc/fabric/api/biome/v1/BiomeModification.java

```diff
@@ -27,7 +27,10 @@
 import net.fabricmc.fabric.impl.biome.modification.BiomeModificationImpl;
 
 /**
- * <b>Experimental feature</b>, may be removed or changed without further notice.
+ * Provides methods for modifying biomes. To create an instance, call
+ * {@link BiomeModifications#create(Identifier)}.
+ *
+ * <p><b>Experimental feature</b>, may be removed or changed without further notice.
```

Comment:
Not for this PR: We should really make this stable, I think ill take a look at the modules and figure out what can become stable.

Reply:
I do not think so because Forge's API looks much better. See #3316